packages/datadog-plugin-google-cloud-pubsub/src/producer.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const ProducerPlugin = require('../../dd-trace/src/plugins/producer')
+
+class GoogleCloudPubsubProducerPlugin extends ProducerPlugin {
+  static get name () { return 'google-cloud-pubsub' }
+  static get operation () { return 'request' }
+
+  start ({ cfg, projectId, messages }) {
+    if (cfg.method !== 'publish') return
+
+    const topic = cfg.reqOpts.topic
+    const span = this.startSpan('pubsub.request', { // TODO: rename
```

Comment:
Are these TODOs before merging or in the future?

Reply:
In the future. For producers and consumers, the span name is supposed to end with `.consume` and `.produce` which was not the case historically. This is a breaking change so it can only be done in a major.